### PR TITLE
[Homepage]:Statistics screenshot added to features section.

### LINKED ISSUE #5771: [DOCS] Statistics screenshot added to features on homepage
### Description

Screenshot 2023-04-01 at 21 39 53

### Screenshots

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by Blaize15**
Closes #5771

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots

statistics

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5775"></a>

### CODE REVIEW

File: pages/index.js [hunk 1]
```diff
@@ -124,6 +124,13 @@ export default function Home({ total, today }) {
         "https://user-images.githubusercontent.com/624760/217969126-c31be76a-6682-41a0-9fdf-8a235d1237f7.png",
       imageAlt: "LinkFree screenshot of links section of an example profile",
     },
+    {
+      name: "LinkFree Statistics ",
```
Comment (eddiejaoude) [inline pages/index.js:128]:
```suggestion
      name: "LinkFree Statistics", 
```

File: pages/index.js [hunk 2]
```diff
@@ -124,6 +124,13 @@ export default function Home({ total, today }) {
         "https://user-images.githubusercontent.com/624760/217969126-c31be76a-6682-41a0-9fdf-8a235d1237f7.png",
       imageAlt: "LinkFree screenshot of links section of an example profile",
     },
+    {
+      name: "LinkFree Statistics", 
+      description: "Explore your Linkfree, profile views and URL hits",
+      imageSrc:
+        "https://user-images.githubusercontent.com/624760/229295853-ddafba87-7feb-4562-9867-4346871dd25a.png",
+      imageAlt: "LinkFree screenshot of /account/statistics page",
```
Comment (eddiejaoude) [inline pages/index.js:132]:
```suggestion
      imageAlt: "LinkFree screenshot of account statistics page",
```

File: pages/index.js [hunk 3]
```diff
@@ -124,6 +124,13 @@ export default function Home({ total, today }) {
         "https://user-images.githubusercontent.com/624760/217969126-c31be76a-6682-41a0-9fdf-8a235d1237f7.png",
       imageAlt: "LinkFree screenshot of links section of an example profile",
     },
+    {
+      name: "LinkFree Statistics", 
+      description: "Explore your Linkfree, profile views and URL hits",
```
Comment (eddiejaoude) [inline pages/index.js:129]:
```suggestion
      description: "View details of your LinkFree profile, with views and url clicks,
```

**Review by eddiejaoude — APPROVED** [review]
Great 👍  thank you!

File: pages/index.js [hunk 4]
```diff
@@ -124,6 +124,13 @@ export default function Home({ total, today }) {
         "https://user-images.githubusercontent.com/624760/217969126-c31be76a-6682-41a0-9fdf-8a235d1237f7.png",
       imageAlt: "LinkFree screenshot of links section of an example profile",
     },
+    {
+      name: "LinkFree Statistics", 
+      description: "View details of your LinkFree profile, with views and url clicks,
```
Comment (eddiejaoude) [inline pages/index.js:129]:
```suggestion
      description: "View details of your LinkFree profile, with views and url clicks",
```

**Comment by Blaize15** [comment]
Thanks Eddie! for Reviewing and making the necessary changes 😊Im learning a lot by just opening a `PR`, this is awesome💜

**Pull request merged**